MSON Specification.md

```diff
@@ -439,12 +439,13 @@ A _Wildcard Type Name_ MAY only be used in a _[Type Specification][]_ for _[Memb
 #### 3.5.3 Type Attribute
 Defines extra attributes associated with the implementation of a type.
 
-- `required` - instance of this type is required
-- `optional` - instance of this type is optional (default)
-- `fixed`    - instance of this type structure and values are fixed
-- `nullable` - instance of this type *Value* MAY be unset (e.g. `null` or `nil`). `nullable` may only be used within properties of objects.
-- `sample`   - Alternate way to indicate a _[Value][]_ is a sample. See _[Sample][]_.
-- `default`  - Alternate way to indicate a _[Value][]_ is a default. See _[Default][]_.
+- `required`   - instance of this type is required
+- `optional`   - instance of this type is optional (default)
+- `fixed`      - instance of this type structure and values are fixed. This attribute is inheritable: all internal structures became fixed too.
+- `fixed-type` - instance of this type structure is fixed, value - is not. This attribute is unheritable.
```

Comment:
To me the language is clear but I am a developer so I do understand terms which might not be clear to everybody. And yes `fixed-type` is not propagating to the nested members. The reason being the flexibility I want array of User objects but I don't want to impose constraints on the User object just because I want it in an array, you can specify any constraints on the User object directly.